lms/templates/problem.html

```diff
@@ -36,7 +36,10 @@ <h3 class="hd hd-2 problem-header" tabindex="-1">
     % endif
     % if answer_available:
     <span class="problem-action-button-wrapper">
-        <button class="show problem-action-btn btn-default btn-small"><span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span></button>
+        <button class="show problem-action-btn btn-default btn-small">
+            <span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span>
+            <span class="show-answer-tooltip sr">${_('To get credit, only select after Submit.')}</span>
```

Comment:
@alisan617 Thanks for clarifying the other scenario. Can you also tell me, does the tooltip serve as the screen reader text for the button, or is the screen reader text separate? I think that if tooltip = sr text then we should include "Show Answer" somewhere in the text, and provide a clear explanation of what happens when they click the button. 
@chris-mike Are tooltips generally expected to be simply descriptive? We're trying to provide a lot of guidance/explanation in this tooltip that might be better provided another way?